internal/scripts/scripts_test.go

```diff
@@ -216,11 +218,18 @@ func (s *S) TestScripts(c *C) {
 		if test.hackdir != nil {
 			test.hackdir(c, rootDir)
 		}
+		if test.create == nil {
+			test.create = func(opts *fsutil.CreateOptions) error {
+				_, err := fsutil.Create(opts)
```

Comment:
I think we should have a list of files created like in `extract_test.go`  but actually in this case it is simpler because you can keep the list here and assert that the entries match the one in the `test.result`, no need to create another test attribute.

Reply:
How was this done?

Reply:
Sorry, forgot to leave comments here. This was not done. I only removed the `create` attribute because it seems to be the same function everytime.

About the list of files created like in `extract_test.go`, comparing with `test.result` will not be an exact match. I mean, of course the entry properties would match, but some entries would be absent in the new list. For example, directories like `/foo/` would be skipped in the new list because they are not created by the `Create` function in `scripts.ContentValue`.

---

Please let me know if you still want to maintain a list for the created files. We can of course ignore the directories and match only the files.

Reply:
Let's have a separate list then which is an array of the files passed to `Create`. That will ensure it is being called properly.

Reply:
Added.